search/search_service/__init__.py

```diff
@@ -89,6 +90,9 @@ def create_app(*, config_module_class: str) -> Flask:
     # Health Check
     api.add_resource(HealthcheckAPI, '/healthcheck')
 
+    # New search endpoint
+    api.add_resource(SearchAPI, '/search_v2')
```

Comment:
is there a reason why we are using such naming convention? APIs are usually versioned like that /api/v1/search and api/v2/search so I wouldn't want us to establish different precedence. the resource is still `search`, if the version changes it should go under different root v tag

Reply:
oh I like that actually I'll try that out since I am rewriting documents API and adding a facets API